.github/template_validation_rules/validate_issue_is_present.json

```
@@ -0,0 +1,7 @@
+{
+  "name": "Rule to validate the the template has an issue attached",
+  "action": "comment",
+  "eventJsonPath": "$[?(@.body.match(/Issue Reference.*: #[0-9]{3}/) && @.user.login != 'renovate-bot')]",
```

Issue references may not always be 3 characters long - e.g. #58
Changing this to match one or more digits might be more suitable:
```suggestion
  "eventJsonPath": "$[?(@.body.match(/Issue Reference.*: #[0-9]+/) && @.user.login != 'renovate-bot')]",
```

---

yea! that's correct, I was trying to make it fail. the final version should be + like you are offering.